src/vmm/src/device_manager/mmio.rs

```diff
@@ -133,12 +137,12 @@ impl MMIODeviceManager {
         Ok(())
     }
 
-    /// Register an already created virtio-over-MMIO device to be used via MMIO transport.
+    /// Register a virtio-over-MMIO device to be used via MMIO transport at a specific slot.
     pub fn register_virtio_mmio_device(
```

Comment:
nit: there are a lot of methods with `register` in the title, could we rename them to be more expressive?
for example, 
- `register_mmio_device` -> `insert_mmio_device`
- `register_new_virtio_mmio_device` -> `register_virtio_mmio_device`
- `register_virtio_mmio_device` -> `register_virtio_mmio_device_with_slot`

Reply:
Yeah this needs to be cleaned up at some point.